Formula/dvm.rb

```diff
@@ -0,0 +1,36 @@
+class Dvm < Formula
+  desc "Docker Version Manager"
+  homepage "https://github.com/getcarina/dvm"
+  version "0.6.2"
+  @@commit = "6467884b2e25d080c25aa647bbcbbf8c12dc45bd"
+  url "https://github.com/getcarina/dvm/archive/#{version}.tar.gz"
+  sha256 "57a14fc85c37d30c3c2265eb0f06d2fdc18273306052a5ccc748b54a5c38660c"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    dvmpath = buildpath/"src/github.com/getcarina/dvm"
+    dvmpath.install Dir["{*,.git}"]
+
+    cd dvmpath do
+      system "make", "VERSION=#{version}", "COMMIT=#{@@commit}", "UPGRADE_DISABLED=true"
```

Comment:
Please replace `#{@@commit}` directly with the string. Ideal abstraction and avoiding all repetition is actually not the most important goal in formulae. They are intended to be easy to read by a human, even one not very familiar with Ruby.